### ISSUE TITLE
Can't locate IO/String.pm in @INC

### ISSUE THREAD

**Issue by moewew** [plk/biber#245]
Copied over from https://github.com/plk/biblatex/issues/854

Running the MWE
```
\documentclass{article}

\usepackage[style=authoryear, backend=biber]{biblatex}

\addbibresource{biblatex-examples.bib}

\begin{document}
\cite{sigfridsson}
\printbibliography
\end{document}
```
with
```
pdflatex test
biber --output-format=bblxml test
```
generates
```
 INFO - This is Biber 2.10
 INFO - Logfile is '50-style-authoryear.blg'
 ERROR - Error loading data source package 'Biber::Output::bblxml': 
 Can't locate IO/String.pm in @INC 
 (you may need to install the IO::String module) 
 (@INC contains: /tmp/par-637672/cache-c69eefcf7af45c44dec592684a863c8f49910910/inc/lib 
 /tmp/par-637672/cache-c69eefcf7af45c44dec592684a863c8f49910910/inc 
 CODE(0x2006640) CODE(0x2006838)) 
 at /tmp/par-637672/cache-c69eefcf7af45c44dec592684a863c8f49910910/inc/lib/Biber/Output/bblxml.pm line 14.
 BEGIN failed--compilation aborted at /tmp/par-637672/cache-c69eefcf7af45c44dec592684a863c8f49910910/inc/lib/Biber/Output/bblxml.pm line 14.
 Compilation failed in require at (eval 353) line 1.
 INFO - ERRORS: 1
```

While @sayahna is using Biber 2.10 from  TeXLive 2018 on Linux Mint 19 I could reproduce the same error with Biber 2.12 from vanilla TeX live 2018 on Ubuntu 18.04. On my Windows 10 machine with MikTeX 2.9's Biber 2.12 the call goes through just fine and the output is a sensible `test.bblxml`.

Indeed I found that `<biber cache>\inc\lib\IO` on my Windows machine contained `String.pm`, but that this file is missing on Ubuntu. Could it be that the module is just not packed for the Linux binary?

**Comment by plk** [comment]
Hmm, I've seen strange things before on Linux with not packing some dependencies. Can you see if this happens with the latest 2.13 DEV linux binary? If so, I will re-pack and explicitly include that.

**Comment by sayahna** [comment]
On Thu, Jan 10, 2019 at 11:07 PM plk <notifications@github.com> wrote:

> Hmm, I've seen strange things before on Linux with not packing some
> dependencies. Can you see if this happens with the latest 2.13 DEV linux
> binary? If so, I will re-pack and explicitly include that.
>
Many thanks for looking into this problem. I will check 2.13 as the first
job tomorrow (it is already late night here in India)  and will get back to
you with the results.

Thanks again.

Best regards
-- 
Radhakrishan (രാധാകൃഷ്ണന്‍)
<https://maps.google.com/maps/place?cid=3748532478409855227>

**Comment by moewew** [comment]
I got the same issue with 2.13 dev (`biber-linux_x86_64`) downloaded today.

**Comment by plk** [comment]
Should be corrected now in DEV 2.13 if you can check.

**Comment by moewew** [comment]
With the newly packaged 2.13 the issue is gone. The call works fine with 64bit Linux (on Ubuntu 18.04) and on Windows (x64).
